include/zephyr/drivers/stepper.h

```diff
@@ -408,36 +407,34 @@ static inline int z_impl_stepper_is_moving(const struct device *dev, bool *is_mo
 }
 
 /**
- * @brief Enable constant velocity mode for the stepper with a given velocity
+ * @brief Run the stepper with a given velocity in a given direction
  *
- * @details activate constant velocity mode with the given velocity in micro_steps_per_second.
- * If velocity > 0, motor shall be set into motion and run incessantly until and unless stalled or
- * stopped using some other command, for instance, motor_enable(false).
+ * @details If velocity > 0, motor shall be set into motion and run incessantly until and unless
+ * stalled or stopped using some other command, for instance, motor_enable(false).
  *
  * @param dev pointer to the stepper motor controller instance
  * @param direction The direction to set
- * @param value The velocity to set in steps per second where one step is dependent on the current
- * microstepping resolution:
- * > 0: Enable constant velocity mode with the given velocity in a given direction
- * 0: Disable constant velocity mode
+ * @param velocity The velocity to set in microsteps per second
+ * > 0: Run the stepper with the given velocity in a given direction
+ * 0: Stop the stepper
```

Comment:
since CI is failing and this needs another update, I might as well point out this which I missed before. This is not correctly formatted (see https://builds.zephyrproject.io/zephyr/pr/82162/docs/doxygen/html/group__stepper__interface.html#gae11f52a907649faec67beebf65e00b96)

```suggestion
 * @param velocity The velocity to set in microsteps per second
 *                 - > 0: Run the stepper with the given velocity in a given direction
 *                 - 0: Stop the stepper
```